pkgs/applications/editors/vim/plugins/get-plugins.nix

```diff
@@ -0,0 +1,20 @@
+with import <localpkgs> {};
+let
+  inherit (vimUtils.override {inherit vim;}) buildVimPluginFrom2Nix;
+  inherit (neovimUtils) buildNeovimPluginFrom2Nix;
+
+  generated = callPackage ./pkgs/applications/editors/vim/plugins/generated.nix {
```

Comment:
would this work when you are `cd`ed into the dirctory?

Reply:
it didn't I just fixed it.